src/centrality/betweenness.jl

```diff
@@ -139,7 +139,7 @@ function _accumulate_endpoints!(
     end
 end
 
-function _rescale!(betweenness::Vector{Float64}, n::Int, normalize::Bool, directed::Bool, k::Int)
+function _rescale!(betweenness::Vector{Float64}, n::Integer, normalize::Bool, directed::Bool, k::Int)
```

Comment:
Yes. I think keeping betweenness values as `Float64`s makes sense (at least until you persuade me otherwise, in which case we'll make a new PR.)